spec/classes/init_spec.rb

```diff
@@ -374,9 +371,7 @@
   context 'with use_default_logging_channel set to valid bool <false>' do
     let(:params) { { :use_default_logging_channel => false } }
 
-    # one bind::channel resources is valid because of hiera example data
-    it { should contain_bind__channel('from_hiera_common') }
-    it { should have_bind__channel_resource_count(1) }
+    it { should have_bind__channel_resource_count(0) }
```

Comment:
I have taken care that the hiera_merge examples do not pollute the other tests anymore.